src/components/template/ToDoList/ToDoList.tsx

```diff
@@ -162,14 +156,15 @@ const ToDoListInput = ({type,onChange,onClick,onPressEnter,hide,value}:ToDoListI
         <ToDoListInputsContainer className={hide?"hide":""}>
             <TextInput 
             value={value}
-            placeholder={type !== "search" || type ?"":"Search..."}
-            prefix={type === "search"?<SearchOutlined/>: " "}
-            style={{width:"100%"}}
+            placeholder={type === "home" || type === "search" ?"Search...":" "}
+            prefix={type === "home" || type === "search" ?<SearchOutlined/>:" "}
+            style={{width:"100%", marginRight:"10px",color: "#828282", border:"1px solid #828282"}}
```

Comment:
I think this is the only one remaining color that is not using constants. Please move/use the color constants.